docs/journal/00-introducing-the-journey.md

```diff
@@ -1,118 +1,110 @@
+- A reference architecture.
+
+The reference architecture is designed to test and validate our assumptions, and to keep our guidance honest. 
 
 > To find out what we mean by "IoT solution", see [_What is an IoT solution?_][intro-to-iot].
 
-## Meet Fabrikam
+## The problem space  
 
-Fabrikam is a startup software development company with a dozen employees. The developers all have a background with .NET and Microsoft Azure. They also have a growing interest in technologies such as NodeJS, Hadoop, and Java.
+We've identified some common scenarios and challenges in IoT. We're interested to hear from other people about their experiences. If you've worked on an IoT solution, how well does it map to what we are presenting?
 
-The company vision is to become a "smart-building service provider"; a company that provides "environmental monitoring services" on a contractual basis. Initially, they intend to offer their services to owners of high-end apartment buildings. Once contracted, they will install devices in the apartments to be monitored. These devices will report on temperature, humidity, smoke alarm and other environmental conditions.
+**Managing devices**. This includes provisioning new devices, taking devices offline (for maintenance or retirement), and device configuration.
 
-Fabrikam will use the data collected from the devices to provide various services to the building owners. They imagine services that include the following alerts:
-- Fire
-- Flood (broken pipes, slow leaks, water seepage)
-- Unexpected temperature changes (possibly indicating a broken HVAC, or a more serious situation)
+Devices may need to be provisioned in the field (e.g., installing sensors in a building), using a self-serve model. That is, the technician who installs or services the device can provision it right there in the field, possibly using a mobile app.
 
-Aside from improving the safety in buildings, Fabrikam hope to demonstrate that constant monitoring and adjustment of the environment can lead to saving on utilities costs. In addition, Fabrikam wants to give individual tenants the ability to view
-the current state of their own apartment through a mobile app.
+Part of provisioning is authentication, so that only authorized devices can send data.      
 
-Fabrikam landed their first contact before they even wrote their first line of code.
-Now they now have a few months to roll out a system to production.
+**Ingesting data**. The most basic task of an IoT system is getting data into the back end, reliably and at scale. Considerations include:
 
-## The Journal
+- The expected data rate (how many devices; how many events per device). 
+- Physical network requirements.
+- Message protocols. (HTTP? [AMQP](https://www.amqp.org/)?) 
+- Protocol translation. In some cases, you might need to translate output from the devices into the expected ingestion format. For example, you might have legacy devices that use non-standard protocols.   
```

Comment:
Encoding _could_ be mentioned here. Both for translation or ingestion in general.